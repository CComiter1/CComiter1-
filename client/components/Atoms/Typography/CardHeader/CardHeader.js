import styled from 'styled-components'

export default styled.h3`
  font-family: 'Roboto', sans-serif;
  margin: 0;
  font-size: 1.6rem;
  color: ${props => props.theme.secondaryColor};
`
