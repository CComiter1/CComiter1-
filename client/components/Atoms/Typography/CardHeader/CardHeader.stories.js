import React from 'react'
import { storiesOf } from '@storybook/react'

import CardHeader from './CardHeader'

storiesOf('Atoms/Typography/CardHeader', module)
  .add('ShortTitle', () => <CardHeader>Hello World</CardHeader>)

