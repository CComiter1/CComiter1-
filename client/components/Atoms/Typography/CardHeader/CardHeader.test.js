import React from 'react'
import renderer from 'react-test-renderer'
import 'jest-styled-components'

import CardHeader from './CardHeader'

describe('CardHeader', () => {
  it('renders correctly', () => {
    const component = renderer.create(
      <CardHeader>HEADER TEXT</CardHeader>
    ).toJSON()
    expect(component).toMatchSnapshot()
  })
})
