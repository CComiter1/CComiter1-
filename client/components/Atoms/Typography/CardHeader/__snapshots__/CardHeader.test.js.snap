// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CardHeader renders correctly 1`] = `
.c0 {
  font-family: 'Roboto',sans-serif;
  margin: 0;
  margin-bottom: .3rem;
  text-align: center;
  font-size: 1.6rem;
}

<h3
  className="c0"
>
  HEADER TEXT
</h3>
`;
